src/PowerShell/PowerShellModuleDetector.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Azure.Functions.PowerShellWorker.Utility;
+using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.PowerShell
+{
+    using LogLevel = WebJobs.Script.Grpc.Messages.RpcLog.Types.Level;
+
+    internal class PowerShellModuleDetector
+    {
+        public static bool IsPowerShellModuleLoaded(System.Management.Automation.PowerShell _pwsh, ILogger _logger, string moduleName)
```

Comment:
Naming convention: we use names starting with underscore for class fields only. Parameter names should be in camelCase without underscore characters

Reply:
I will not block the PR on this, but please clean it up, now or later